samza/jobs-common/src/main/java/org/sunbird/jobs/samza/util/TestUtil.java

```diff
@@ -0,0 +1,30 @@
+package org.sunbird.jobs.samza.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class TestUtil {
```

Comment:
If it is TestUtil then shouldn't it be part of 'test' folder rather than 'main' folder?